app/src/main/res/values/strings.xml

```diff
@@ -287,4 +287,6 @@
   <string name="wikidata_edit_failure">Failed to update corresponding Wikidata entity!</string>
   <string name="menu_set_wallpaper">Set wallpaper</string>
   <string name="wallpaper_set_successfully">Wallpaper set successfully!</string>
+
+  <string name="user_not_logged_in">No active session found for this user.</string>
```

Comment:
Probably more user-friendly to say something like: "Login expired, please login again" ?